ui/qt-lib/src/Ariadne/UI/Qt/Face.hs

```diff
@@ -210,3 +205,5 @@ data UiAddressInfo = UiAddressInfo
 data UiSelectionInfo
   = UiSelectionWallet !UiWalletInfo
   | UiSelectionAccount !UiAccountInfo
+
+type MnemonicGenerator = Int -> IO [Text]
```

Comment:
AFAIU, this module exports various stuff related to *what UI can do*, not *what UI needs*. So I think this alias should be somewhere in another place (maybe in `WalletTree.hs`).